### PR TITLE
Add parsed command dict db col

### LINKED ISSUE #927: Add parsed aerie-ampcs command dictionary JSON to command-dictionary table
When we upload a command dictionary we need to add the [parsed JSON](https://github.com/NASA-AMMOS/aerie/blob/develop/sequencing-server/src/app.ts#L117) to a new column in the [command_dictionary table](https://github.com/NASA-AMMOS/aerie/blob/develop/sequencing-server/sql/sequencing/tables/command_dictionary.sql).
Needed before we can make progress on https://github.com/NASA-AMMOS/aerie-ui/issues/630.

### PR DESCRIPTION
**Pull request by cohansen**
* **Tickets addressed:** Closes #927
* **Review:** By commit
* **Merge strategy:** Merge (no squash)

## Description
Adds a new column in the `command_dictionary` table that stores the entire parsed command dictionary. This is in support of hooking into `min` and `max` values that @Cobular is working on. 

## Verification
Added a basic unit test to make sure the command dictionary gets inserted.

## Documentation
NA

## Future work
NA

### CODE REVIEW

File: sequencing-server/test/command-dictionary.spec.ts [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+beforeEach(async () => {
+  graphqlClient = new GraphQLClient(process.env['MERLIN_GRAPHQL_URL'] as string, {
+    headers: { 'x-hasura-admin-secret': process.env['HASURA_GRAPHQL_ADMIN_SECRET'] as string },
+  });
+});
+
+describe('upload command dictionary', () => {
+  it('should upload a command dictionary and all of the fields should be populated correctly', async () => {
+    expect(await insertCommandDictionary(graphqlClient)).toBeDefined();
+  }, 30000);
```
Comment (Mythicaeda) [inline sequencing-server/test/command-dictionary.spec.ts]: Could you please explain how the "and all of the fields should be populated correctly" part of the test name is asserted?

Comment (cohansen) [reply sequencing-server/test/command-dictionary.spec.ts]: I missed the rest of the test changes, it does a strict equality check from the `parsed_json` column to parsing our sample command dictionary on the fly.

File: sequencing-server/src/app.ts [hunk 2]
```diff
@@ -123,17 +123,18 @@ app.post('/put-dictionary', async (req, res, next) => {
   logger.info(`command-lib generated - path: ${commandDictionaryPath}`);
 
   const sqlExpression = `
-    insert into command_dictionary (command_types_typescript_path, mission, version)
-    values ($1, $2, $3)
+    insert into command_dictionary (command_types_typescript_path, mission, version, parsed_json)
```
Comment (goetzrrGit) [inline sequencing-server/src/app.ts:126]: I know that @Mythicaeda is going to drop the `update_at` column, but I think you guys are working that out.

File: sequencing-server/test/command-expansion.spec.ts [hunk 3]
```diff
@@ -37,7 +37,7 @@ beforeEach(async () => {
     simulation_start_time: '2020-001T00:00:00Z',
     simulation_end_time: '2020-002T00:00:00Z',
   });
-  commandDictionaryId = await insertCommandDictionary(graphqlClient);
+  commandDictionaryId = (await insertCommandDictionary(graphqlClient)).id;
```
Comment (goetzrrGit) [inline sequencing-server/test/command-expansion.spec.ts:40]: Just curious why we are getting the `id` now. Was the previous code getting the object but never the id?

**Review by goetzrrGit — APPROVED** [review]
No issues just a couple of questions.

File: sequencing-server/src/app.ts (hunk 2 above)
Comment (cohansen) [reply sequencing-server/src/app.ts:126]: Yeah, it's part of another PR!

File: sequencing-server/test/command-expansion.spec.ts (hunk 3 above)
Comment (cohansen) [reply sequencing-server/test/command-expansion.spec.ts:40]: Previous code was just getting the id, now I need a few of the columns so I'm specifically asking for the id where we used to get it ad the default option.

**Pull request merged**